src/components/VaultRow/lib/VaultDepositForm.tsx

```diff
@@ -159,6 +159,31 @@ export const VaultDepositForm: FC<VaultDepositWithdrawProps> = ({
     onSuccess: () => onDepositSuccess(),
   })
 
+  // DEBUG HERE
```

Comment:
I think we can prob remove this now right?

Reply:
yes